config/doctrine/User.orm.yml

```diff
@@ -51,6 +51,9 @@ App\Entity\User:
     mailCryptPublicKey:
       type: text
       nullable: true
+    wkdKey:
+      type: text
+      nullable: true
```

Comment:
See the comment above about binary field types, especially `blob`
https://www.doctrine-project.org/projects/doctrine-dbal/en/2.11/reference/types.html#blob